Dockerfile

```diff
@@ -16,8 +16,9 @@ COPY Pip* /app/
 
 RUN pip install --upgrade pip && \
     pip install pipenv && \
-    pipenv install --dev --system --skip-lock
-
+    pipenv install --dev --system --skip-lock && \
```

Comment:
Please revert this file, you can use default python lib: https://docs.python.org/3/library/csv.html

mongoengine is already in `Pipfile`

Reply:
Reverted